legion_test/legion_test/robot/utils.py

```diff
@@ -138,19 +138,24 @@ def find_model_information_in_edi(parsed_edi_output, model_id, model_version=Non
         return founded[0]
 
     @staticmethod
-    def check_valid_http_response(url):
+    def check_valid_http_response(url, token=None):

```

Comment:
Function should be renamed to show that it can be used only for model invocation